packages/dev/biome-config/package.json

```diff
@@ -16,9 +14,7 @@
         "url": "git://github.com/lokalise/shared-ts-libs.git"
     },
     "exports": {
-        "./*": [
-            "./configs/*.jsonc"
-        ]
+        "./*": ["./configs/*.jsonc"]
```

Comment:
`exports` value can be a plain string instead of an array of strings

Reply:
fair, but it's more flexible this way